packages/shared/components/table/components/filtering/event-type-filter/event-type-filter.constants.js

```diff
@@ -11,8 +11,29 @@ export const EVENTS_TYPES = {
   Ended: "Ended"
 };
 
-export const EVENT_TYPE_FILTER_VALUES = [...Object.keys(EVENTS_TYPES)].map(
-  x => ({ value: x, label: x })
-);
+export const MANAGER_EVENT_TYPE = {
+  All: "All",
+  AssetStarted: "AssetStarted",
+  ProgramPeriodStats: "ProgramPeriodStats",
+  ProgramPeriodEnds: "ProgramPeriodEnds",
+  InvestorInvest: "InvestorInvest",
+  InvestorWithdraw: "InvestorWithdraw",
+  ManagerInvest: "ManagerInvest",
+  ManagerWithdraw: "ManagerWithdraw",
+  AssetFinished: "AssetFinished",
+  EntranceFee: "EntranceFee",
+  ExitFee: "ExitFee"
+};
```

Comment:
Maybe we should replace them by enum in project?
This will make it possible to use such constants in types.

Reply:
Ok